apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -526,12 +570,19 @@ export function StudentAssessmentInstance({
                         </li>
                         <li>
                           Look at <strong>Status</strong> to confirm that each question has been
-                          graded. Questions with <strong>Available points</strong> can be attempted
-                          again for more points. Attempting questions again will never reduce the
-                          points you already have.
+                          ${someQuestionsForbidRealTimeGrading
```

Comment:
Should this be `saved (gradable)`?

Reply:
I don't think so.

When real-time grading is globally enabled on an assessment, we don't say "make sure everything is saved or graded", even though "saved" is a valid state in which to leave something at the end of an assessment, since it will be automatically graded when the assessment is closed. We always want students to be driving towards having graded everything themselves because we generally think that's best for them, especially when multiple attempts are in play.

Via the same line of thinking, with a mix of real-time and non-real-time grading, students should always end up having clicked "grade" wherever possible, which would leave them with a mixture of "saved" and some form of a "graded" status.

Reply:
The `ExamQuestionStatus` should make that clear. Note that there's no status literally called "graded".